moveit_core/robot_state/test/robot_state_test.cpp

```diff
@@ -579,6 +579,7 @@ std::size_t generateTestTraj(std::vector<std::shared_ptr<robot_state::RobotState
 TEST_F(OneRobot, testGenerateTrajectory)
 {
   const robot_model::JointModelGroup* joint_model_group = robot_model_->getJointModelGroup("base_from_base_to_e");
+  EXPECT_TRUE(joint_model_group);
```

Comment:
This should be an assert, because the subsequent code relies on the joint model group to be well-defined.
```suggestion
  ASSERT_TRUE(joint_model_group);
```